### PR TITLE
Crazy Quiz - extension quiz with progress tracker

### PR DESCRIPTION
**Pull request by riittagerhards**
Add progess tracking for questions, fix minor naming issues.

image

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -57,17 +61,25 @@ function setNewQuestion() {
...
   myQuestion.textContent = questionList[questionIndex];
   correctAnswer = answerList[questionIndex];
-  document.body.removeChild(document.querySelector(".correct"));
+  sectionQuiz.removeChild(document.querySelector(".correct"));
+  updateProgress();
   quizEnd();
 }
 function quizEnd() {
-  if (questionIndex === 4) {
+  if (questionIndex === questionList.length) {
```
Comment (AnkePfei2fer) [inline main.js:69]: You could think of adding an `if` condition to `setNewQuestion` and setting `else` to `quizEnd`

**Review by AnkePfei2fer — COMMENTED** [review]
lgtm, just one suggestion

File: main.js (hunk 1 above)
Comment (riittagerhards) [reply main.js:69]: Good Idea, thanks!

**Comment by riittagerhards** [comment]
Resolv the issue in next pull request

**Pull request merged**